lib/web3/property.js

```diff
@@ -76,7 +81,7 @@ Property.prototype.extractCallback = function (args) {
  */
 Property.prototype.attachToObject = function (obj) {
     var proto = {
-        get: this.get.bind(this),
+        get: this.buildGet() 
```

Comment:
I don't see the advantage of moving this into its own function, if you simply write:
```js
var proto = {
        get: function get() {
              return property.formatOutput(property.requestManager.send({
                  method: property.getter
             })); 
        }
}
```